test/mitmproxy/proxy/test_mode_servers.py

```diff
@@ -318,6 +318,33 @@ async def test_udp_dual_stack(caplog_async):
         assert await caplog_async.await_log("stopped")
 
 
+@pytest.mark.parametrize("transport_protocol", ["udp", "tcp"])
+async def test_dns_start_stop(caplog_async, transport_protocol):
```

Comment:
Is there a specific motivation for this test?

(I'm mostly asking because DnsInstance doesn't do anything special, and tests that do socket I/O are expensive in terms of runtime)

Reply:
I added this test to check if DNS is listening on both TCP and UDP ports

Reply:
Fair enough, thanks!